packages/x-date-pickers/package.json

```diff
@@ -55,6 +56,8 @@
     "rifm": "^0.12.1"
   },
   "peerDependencies": {
+    "@emotion/react": "^11.9.0",
+    "@emotion/styled": "^11.8.1",
```

Comment:
Outside of the scope that I think should have been as low as possible, I would personally assume that this change is going in the wrong direction: #32668.